synapse/models/infotech.py

```diff
@@ -69,6 +212,22 @@ def getBaseModels():
                     'optfields': 'str=it:dev:str,int=it:dev:int,bytes=file:bytes',
                     'doc': 'A Windows registry key/value pair.'}),
 
+                ('it:semver', {
+                    'ctor': 'synapse.models.infotech.SemverType',
+                    'doc': 'Semantic Version type.'}),
+                ('it:prod:soft', {
+                    'subof': 'guid',
+                    'doc': 'A arbitrary, unversioned software product.'}),
+                ('it:prod:softver',
+                 {'subof': 'guid',
+                  'doc': 'A version of a particular software product.'}),
+                ('it:hostsoft', {'subof': 'comp',
+                                 'fields': 'host,it:host|softver,it:prod:softver',
+                                 'doc': 'A version of a software product which is present on a given host.'}),
+                # ('it:prod:hard', {'subof': 'guid', 'doc': ''}),
```

Comment:
I'd remove these comments but track the future work.

Reply:
fixed and backlogged